Main.java

```diff
@@ -9,7 +10,17 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    //// Creates a Scanner used for input
+   Scanner input = new Scanner(System.in);
+
+    System.out.println("Please enter in two integers, on seperate lines, to divide.");
+    // declare and initlize the integers
+   
+ 
+
+
+ // declare and calculate variable for integer
+ System.out.println("10/3 is 3 with a remainder of 1.");
+ 
```

Comment:
Again, this is a hard coded answer. We need to get two numbers from the user. Once we have these two numbers, we can calculate the quotient by doing some division and we can get the remainder by doing the modulus statement. Once we have those 2 pieces, we will output those answers to the user.